### PR TITLE
[ch205] Fix display of potential advertisers on property

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: lib/code_fund/properties/properties.ex [hunk 1]
```diff
@@ -284,12 +283,17 @@ defmodule CodeFund.Properties do
     do: Repo.get_by!(Property, name: name) |> Repo.preload([:user, :audience])
 
   def get_all_display_rates(%Property{} = property) do
-    AdService.Query.ForDisplay.build(
-      programming_languages: property.programming_languages,
-      topic_categories: property.topic_categories
-    )
-    |> CodeFund.Repo.all()
-    |> AdService.Math.Basic.get_all_display_rates()
+    audience = property.audience
```
Comment (StareIntoTheBeard) [inline lib/code_fund/properties/properties.ex]: let's match that out in the function def if we can

File: lib/code_fund/properties/properties.ex [hunk 2]
```diff
@@ -284,12 +283,17 @@ defmodule CodeFund.Properties do
...
+
+    case audience do
+      nil ->
+        []
+
+      %Audience{} = audience ->
+        AdService.Query.ForDisplay.build(audience, nil)
```
Comment (StareIntoTheBeard) [inline lib/code_fund/properties/properties.ex]: If the old `build` function based on programming languages and topic categories isn't called any more, can we rip out all of that unused logic/function calls?

Comment (coderberry) [reply lib/code_fund/properties/properties.ex]: Yes

File: lib/code_fund/properties/properties.ex (hunk 1 above)
Comment (coderberry) [reply lib/code_fund/properties/properties.ex]: Sounds good

**Pull request merged**